src/server.test.ts

```diff
@@ -23,9 +38,21 @@ test('POST /scan/configure', done => {
     })
 })
 
-test('POST /scan/scan', done => {
+test('POST /scan/scanBatch', done => {
+  mockScanner.doScan.mockResolvedValue('')
+  request(app)
+    .post('/scan/scanBatch')
+    .set('Accept', 'application/json')
+    .expect(200, { status: 'ok' })
+    .then(() => {
+      expect(scanner.doScan).toBeCalled()
+      done()
+    })
+})
+
+test('POST /scan/invalidateBatch', done => {
```

Comment:
The last step in this test expects `scanner.doScan` to have been called, but the code doesn't call `doScan`:

```
app.post('/scan/invalidateBatch', (_request: Request, response: Response) => {
  response.json({ status: 'ok' })
})
```

Reply:
weird, maybe it was succeeding from the other test in the file? In any case, removed for now.